src/DynamoCoreWpf/Views/GuidedTour/RealTimeInfoWindow.xaml.cs

```diff
@@ -58,5 +58,18 @@ private void CloseButton_Click(object sender, System.Windows.RoutedEventArgs e)
         {
             CleanRealTimeInfoWindow();
         }
+
+        /// <summary>
+        /// This method will update the current location of the RealTimeInfo window due that probably the window was moved or resized
+        /// </summary>
+        public void UpdateLocation()
+        {
+            if (IsOpen == true)
+            {
+                //Uodating the Offset will produce that the Popup will update the position(that's the only way I found).
+                HorizontalOffset = HorizontalOffset + 1;
+                HorizontalOffset = HorizontalOffset - 1;
```

Comment:
same comments here, any alternative?

Reply:
@QilongTang I changed the way the popup updates their locations, now I'm using reflection (the UpdatePosition() method in the Popup class is private), please let me know if this is ok since is the only way I found to do it.
https://github.com/DynamoDS/Dynamo/pull/11959/commits/c923ee41f84e324dd8d71b133e31219db5804ae7